### PR TITLE
Metrics chart

### PR DESCRIPTION
**Pull request by jmckulk**
This change adds a helm chart to install the Prometheus and Grafana deployments. 
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:
[ch8640]

### CODE REVIEW

File: docs/content/installation/other/helm.md [hunk 1]
```diff
@@ -176,6 +176,33 @@ pgo client version 4.4.0-beta.1
 pgo-apiserver version 4.4.0-beta.1
 ```
 
+## Metrics Chart
+
+The PostgreSQL Operator metrics infrastructure can be installed using a separate
+Helm chart. This chart can install or uninstall metrics in a namespace where
+the operator is currently running. The metrics installer runs similarly to the
```
Comment (andrewlecuyer) [inline docs/content/installation/other/helm.md:183]: > This chart can install or uninstall metrics in a namespace where
> the operator is currently running

Couldn't it be installed in a separate namespace as well, e.g. if `metrics_namespace` has a different namespace (or the Helm artifacts themselves if another namespace is specified when running `helm install metrics`.

Comment (jmckulk) [reply docs/content/installation/other/helm.md:183]: Yeah that is true. What about this?
> This chart can install or uninstall metrics in a provided namespace.

File: docs/content/installation/other/helm.md [hunk 2]
```diff
@@ -176,6 +176,33 @@ pgo client version 4.4.0-beta.1
...
+similarly to the `postgres-operator` chart by creating the ServiceAccount, RBAC
+resources, and ConfigMap that are needed to run the install job.
+
+The `metrics` Helm chart is available in the [Helm](https://github.com/CrunchyData/postgres-operator/tree/master/installers/helm)
+directory in the PostgreSQL Operator repository.
+
+### Installing
+
+{{% notice tip %}}
+Ensure that you have install the PostgreSQL Operator before installing metrics.
```
Comment (cbandy) [inline docs/content/installation/other/helm.md:193]:
```suggestion
Ensure that you have installed the PostgreSQL Operator before installing metrics.
```

File: docs/content/installation/other/helm.md (hunk 1 above)
Comment (jmckulk) [reply docs/content/installation/other/helm.md]: I ended up going with something else entirely when i committed the change
>This chart can install or uninstall the Grafana and Prometheus deployments that are part of the metrics stack.

**Pull request merged**